lib/commands/livesync.ts

```diff
@@ -0,0 +1,26 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+export class UsbLivesyncCommand implements ICommand {
```

Comment:
There is no need to prefix this with `Usb`. Both iOS and Android support this over the air.